react/features/authentication/middleware.js

```diff
@@ -108,9 +108,12 @@ MiddlewareRegistry.register(store => next => action => {
     case CONNECTION_FAILED: {
         const { error } = action;
 
-        error
+        if (error
             && error.name === JitsiConnectionErrors.PASSWORD_REQUIRED
-            && store.dispatch(_openLoginDialog());
+            && typeof error.recoverable === 'undefined') {
+            error.recoverable = true;
+            store.dispatch(_openLoginDialog());
```

Comment:
doesn't this prevent the dialog from being re-shown if the wrong credentials are entered?

Reply:
Ack, leave it then.